gp-includes/schema.php

```diff
@@ -68,15 +70,15 @@ function gp_schema_get() {
 	 */
 
 	/*
- 	 * The maximum length for the slug component of the project_id_slug_locale key is limited by the index size limit
+	  * The maximum length for the slug component of the project_id_slug_locale key is limited by the index size limit
```

Comment:
Only one whitespace after the tab.

Reply:
Fixed in https://github.com/GlotPress/GlotPress-WP/pull/1076/commits/aaf1f070efcc5cb7f040b2d812355ad72d5cc83d